cli/aws_orbit/data/init/default-env-manifest.yaml

```diff
@@ -1,33 +1,39 @@
 Name: ${name}
-Region: ${region}
-CodeartifactDomain: aws-orbit
-CodeartifactRepository: python-repository
-ScratchBucketArn: !SSM ${/orbit/dev-env/demo::ScratchBucketArn}
-UserPoolId: !SSM ${/orbit/dev-env/demo::UserPoolId}
-SharedEfsFsId: !SSM ${/orbit/dev-env/demo::EFSFilesystemID}
-SharedEfsSgId: !SSM ${/orbit/dev-env/demo::SharedEFSSecurityGroup}
+ScratchBucketArn: !SSM ${/orbit-f/${name}/resources::ScratchBucketArn}
+UserPoolId: !SSM ${/orbit-f/${name}/resources::UserPoolId}
+SharedEfsFsId: !SSM ${/orbit-f/${name}/resources::SharedEfsFsId}
+SharedEfsSgId: !SSM ${/orbit-f/${name}/resources::SharedEfsSgId}
+Networking:
+    VpcId: !SSM ${/orbit-f/${name}/resources::VpcId}
+    PublicSubnets: !SSM ${/orbit-f/${name}/resources::PublicSubnets}
+    PrivateSubnets: !SSM ${/orbit-f/${name}/resources::PrivateSubnets}
+    IsolatedSubnets: !SSM ${/orbit-f/${name}/resources::IsolatedSubnets}
+    Data:
+        InternetAccessible: !ENV ${INTERNET_ACCESSIBLE::true}
+        NodesSubnets: !SSM ${/orbit-f/${name}/resources::NodesSubnets}
+    Frontend:
+        LoadBalancersSubnets: !SSM ${/orbit-f/${name}/resources::LoadBalancersSubnets}
+        SslCertArn: !SSM ${/orbit-f/${name}/resources::SslCertArn}
 Images:
     JupyterUser:
-        Repository: aws-orbit-jupyter-user
-        Source: public
+        Repository: ${account_id}.dkr.ecr.${region}.amazonaws.com/orbit-${name}/jupyter-user
         Version: latest
-    JupyterUserSpark:
-        Repository: aws-orbit-jupyter-user-spark
-        Source: public
+    OrbitController:
+        Repository: ${account_id}.dkr.ecr.${region}.amazonaws.com/orbit-${name}/orbit-controller
```

Comment:
In few manifests, i have noticed `!ENV ${ACCOUNT_ID::}` and here `${account_id}`. Can we make it consistent?

Reply:
No...this is not using the replace function with the env variables....this is specific to this code...there is no !ENV